test/support/supportedExtensions.ts

```diff
@@ -0,0 +1,277 @@
+/* tslint:disable max-line-length */
```

Comment:
@JimiC, the thing here is to control the inputs of the function. If I use the original file I cannot be certain of what inputs it will receive. If the structure changes then these "duplicated" files should have to change accordingly and, obviously, the tests, too. It's a task that anyway the dev modifying the code will have to do in order to tests his changes. I don't see it as a problem.

Regarding the second point, you're right. 👍